apps/web/src/utils/shortUrlSlug.ts

```diff
@@ -0,0 +1,9 @@
+import { customAlphabet } from 'nanoid';
+
+// removed confusing characters from the alphabet (0, O, I, l) for the URL
+const alphabet = 'ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz123456789';
+
+export function newShortURLSlug(): string {
```

Comment:
we can probably just colocate this with the slug stuff? idk if anyone else will be using this

Reply:
Noted!! Thanks.